src/networking.c

```diff
@@ -1392,7 +1395,9 @@ void freeClientArgv(client *c) {
     c->cmd = NULL;
     c->argv_len_sum = 0;
     c->argv_len = 0;
-    zfree(c->argv);
+    if(c->argv != c->argv_static) {
+        zfree(c->argv);
+    }
     c->argv = NULL;
```

Comment:
c->argv should be reset to equal c->argv_static; and in createClient function, the init value of c->argv also should be argv_static